src/commands/interface-manager/MatrixPromptForAccept.tsx

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (C) 2023 Gnuxie <Gnuxie@protonmail.com>
+ * All rights reserved.
+ */
+
+import { renderMatrixAndSend } from "./DeadDocumentMatrix";
+import { BaseFunction, InterfaceCommand } from "./InterfaceCommand";
+import { JSXFactory } from "./JSXFactory";
+import { MatrixContext } from "./MatrixInterfaceAdaptor";
+import { PromptResponseListener } from "./MatrixPromptUX";
+import { ParamaterDescription } from "./ParamaterParsing";
+import { PromptOptions } from "./PromptForAccept";
+import { CommandResult } from "./Validation";
+
+// How are prompts going to work?
+// We need to temporarily hook onto the event emitter.
+async function promptDefault<PresentationType>(this: MatrixContext, paramater: ParamaterDescription, command: InterfaceCommand<BaseFunction>, defaultPrompt: PresentationType) {
+    await renderMatrixAndSend(
+        <p>
+            No argument was provided for the paramater {paramater.name}, would you like to accept the default?<br/>
+            {defaultPrompt}
+        </p>,
+        this.roomId, this.event, this.client
+    )
+}
+
+// FIXME: <ol> raw tags will not work if the message is sent across events.
+//        If there isn't a start attribute for `ol` then we'll need to take this into our own hands.
```

Comment:
The attribute is called `start`. Not sure how to apply this in the incremental renderer without looking. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ol#attr-start

Reply:
Maybe would require follow up, but to do this what you would need is: the fringe walker in `DeadDocument.ts` would have to borrow the dynamic environment from -- the same file, and it already does. So then, it would have to bind the item count when there is an `ol` or at least read it? 

Since i think the markdown renderer and html renderer both rely on this variable. So then you'll probably also have to make a common set of variables for handling ordered lists.